packages/botonic-dx-bundler-rspack/README.md

```diff
@@ -0,0 +1,31 @@
+# Botonic bundler Rspack
+
+## What Does This Package Do?
+
+This package simplifies the configuration of a rspack bundler to build a Botonic project.
+
+## IMPORTANT
+
+- Webviews imported inside the webviews/index.ts must be classes or functions but cannot be arrow functions.
```

Comment:
I would put an example and explain the why briefly.
For example:
Do:
```
export function MyWebview(){
...
return
}
```
instead of:
```
export const MyWebview = () => {
...
return
}
```

This is because the bundler does not preserve correctly Webview names on transpilation process.